byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -14,21 +14,42 @@
 
 package com.google.maps;
 
-import com.google.maps.model.LatLng;
-import com.google.maps.model.PlaceType;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.maps.model.PlaceType;
 import java.io.IOException;
+import java.lang.reflect.Type;
 import java.util.ArrayList;
+import javax.servlet.annotation.MultipartConfig;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns all PlaceTypes from Places API. */
+
+/**
+ * Servlet that returns all PlaceTypes from Places API.
+ * Stores which placeTypes, or interests, user selects
+ * in datastore with their specific Trip Entity.
+ */
+@MultipartConfig
```

Comment:
Note: I think you only use MultipartConfig when POST requests will be using FormData. If just sending json data, I don't think you need this.

Reply:
Removed